Userland/Libraries/LibMedia/CMakeLists.txt

```diff
@@ -13,43 +13,37 @@ set(SOURCES
     VideoFrame.cpp
 )
 
-if (HAVE_PULSEAUDIO)
-    list(APPEND SOURCES
-        Audio/PlaybackStreamPulseAudio.cpp
-        Audio/PulseAudioWrappers.cpp
-    )
-endif()
-
-if (HAS_FFMPEG)
-    list(APPEND SOURCES
-        Audio/FFmpegLoader.cpp
-        FFmpeg/FFmpegVideoDecoder.cpp
-    )
-else()
-    list(APPEND SOURCES FFmpeg/FFmpegVideoDecoderStub.cpp)
-endif()
-
 serenity_lib(LibMedia media)
 target_link_libraries(LibMedia PRIVATE LibCore LibCrypto LibRIFF LibIPC LibGfx LibThreading LibUnicode)
 
-if (HAS_FFMPEG)
+if (NOT ANDROID)
+    target_sources(LibMedia PRIVATE
+            Audio/FFmpegLoader.cpp
+            FFmpeg/FFmpegVideoDecoder.cpp
+    )
     target_link_libraries(LibMedia PRIVATE PkgConfig::AVCODEC PkgConfig::AVFORMAT PkgConfig::AVUTIL)
+else()
+    # FIXME: Need to figure out how to build or replace ffmpeg libs on Android
+    target_sources(LibMedia PRIVATE FFmpeg/FFmpegVideoDecoderStub.cpp)
 endif()
 
+# Audio backend -- how we output audio to the speakers
 if (HAVE_PULSEAUDIO)
+    target_sources(LibMedia PRIVATE
+        Audio/PlaybackStreamPulseAudio.cpp
+        Audio/PulseAudioWrappers.cpp
+    )
     target_link_libraries(LibMedia PRIVATE pulse)
     target_compile_definitions(LibMedia PRIVATE HAVE_PULSEAUDIO=1)
-endif()
-
-if (APPLE AND NOT IOS)
+elseif (APPLE AND NOT IOS)
     target_sources(LibMedia PRIVATE Audio/PlaybackStreamAudioUnit.cpp)
 
     find_library(AUDIO_UNIT AudioUnit REQUIRED)
     target_link_libraries(LibMedia PRIVATE ${AUDIO_UNIT})
-endif()
-
-if (ANDROID)
+elseif (ANDROID)
     target_sources(LibMedia PRIVATE Audio/PlaybackStreamOboe.cpp)
     find_package(oboe REQUIRED CONFIG)
     target_link_libraries(LibMedia PRIVATE log oboe::oboe)
+else()
+    message(FATAL_ERROR "No audio backend available")
```

Comment:
Maybe fine having this be a warning instead? It just makes bringing up new platforms easier when we can defer handling audio. There's a runtime check in WebContent/main.cpp to throw an error if we try to create an `<audio>` element with no backend support.
See: https://github.com/LadybirdBrowser/ladybird/commit/553d35e5039e7f0add5caefeb0b7ee3d7bbe82fb